src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.DefaultProperties.targets

```diff
@@ -74,7 +74,7 @@
     <UseNativeHttpHandler Condition="'$(UseNativeHttpHandler)' == ''">true</UseNativeHttpHandler>
 
     <!-- XA feature switches defaults -->
-    <XATypeRegistrationFallback Condition="'$(XATypeRegistrationFallback)' == ''">false</XATypeRegistrationFallback>
+    <VSAndroidDesigner Condition="'$(VSAndroidDesigner)' == ''">false</VSAndroidDesigner>
```

Comment:
So will the designer have to change anything on their side to enable this property?

Reply:
They would need to set the `$(VSAndroidDesigner)` to `true`. Or alternatively we might set it to false only for release configuration?

Reply:
The designer works fine for Release builds, so I don't think we can do it there.

For now, maybe you could file a quick issue on what they'll need to do? I'll find the right person take a look later on.

Reply:
I looked closer at it and the msbuild property only affects linking. So designer should run fine with `Debug` configuration.

To disable the fallback during runtime, it would need to be explicitely set in code by calling `AppContext.SetSwitch` or in the application configuration file. So designer should be fine.